workflow-trace-viewer/src/jvmMain/kotlin/com/squareup/workflow1/traceviewer/model/Node.kt

```diff
@@ -31,6 +31,20 @@ internal data class Node(
   override fun hashCode(): Int {
     return id.hashCode()
   }
+
+  companion object {
+    fun getNodeFields(): List<String> {
+      return listOf("Props", "State")
+    }
```

Comment:
This can just be a val, and i'd just statically allocate it.
```suggestion
    val nodeFields: List<String> = listOf("Props", "State")
```

Reply:
Done